book/website/community-handbook/onboarding-offboarding/onboarding-core-team.md

```diff
@@ -0,0 +1,64 @@
+# ☀️ Welcome to *The Turing Way* Core Team!☀️
+
+:::success
+This onboarding document is a living document – as is *The Turing Way* itself. As you'll learn from being in this community: this is a constantly-evolving project that holds [emergence as a core value](https://hackmd.io/kVkwKSYuSLWWhLzlNSuT0g). In other words: as the project evolves – so will the team that enables its sustainability as a project.
+
+From April to August 2022, Anne (TTW CM, 2022) documented [documented her research of/within *The Turing Way* community on Github](https://github.com/alan-turing-institute/the-turing-way/discussions/2552). Much of that research has been used to make this onboarding document.
+:::
+
+[TOC]
+
+# TTW Core Team Onboarding checklists
+- [ ] Review [TTW Onboarding document](https://hackmd.io/pBysnLn2QxKMCHqPKvj9uA?both) (this document!)
+- [ ] Review [TTW Offboarding document](https://hackmd.io/@turingway/ttw-core-team-offboarding) (not to be used for now!)
+
+## Accounts and Permissions
+- [ ] Add to #ttw-core-team channel on Slack
+- [ ] Add to [TTW Github repository](https://github.com/alan-turing-institute/the-turing-way) with "Maintainer" permissions (change as needed for role)
+- [ ] Add to [Ways of Working](https://github.com/alan-turing-institute/the-turing-way/blob/main/ways_of_working.md) page on TTW Github repository 
+- [ ] Share [zenodo link for The Turing Way community](https://zenodo.org/communities/the-turing-way/?page=1&size=20) 
+- [ ] Add to [Core Team Google group](https://groups.google.com/u/1/g/core-team-2022) (currently dormant)
+- [ ] (If applicable) Add to HackMD team for The Turing Way
+- [ ] (If applicable) Give Accounts/Password Access on Keybase
+- [ ] (If applicable) Add directly to [The Turing Way Core Google Drive](https://drive.google.com/drive/u/0/folders/1bdjA-fBotMi9CnjDqu84d5hPZWwjgi-x)
+    - [ ] Share [promotion pack](https://drive.google.com/drive/u/1/folders/1mzGmbJkPnP5q1goQesxDc_E5zAPL0eTF) folder
+    - [ ] Share [conference talks/workshop applications](https://drive.google.com/drive/u/0/folders/1eNHq9sXq6Z8Q5Ep0MzNS0GsB3FKgTi3E) folder
+- [ ] (If applicable) Add to Canva team
+- [ ] (If applicable) Add to Penpot team
+- [ ] (If applicable) Add to Keybase group
+
+## About yourself
+- [ ] Add your information to [TTW Team slides](https://docs.google.com/presentation/d/1PeQ_gg5tOJVhOA4aSnVumsowD-bfRFEroWfGsxxaGCw/edit#slide=id.g1aee8e18598_0_0)
```

Comment:
Does this apply to newcomers outside the Turing?

Reply:
Historically, yes - this has!